modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -15,6 +15,17 @@ If you reuse the `BareMetalHost` object definition from an existing control plan
 Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by the {product-title} installation program.
 ====
 
+.Prerequisites
+
```

Comment:
Can you also add a comment to make sure the baremetal operator, metal3, image cache, etc pods in openshift-machine-api namespace are all in running state along with openshift API access.

Reply:
Thanks for the feedback. If this is necessary, should we add this as a first step of the procedure, before the user starts to delete the machine and bmh objects?

Would the command to verify the state of these Operators be: 

oc get clusteroperators 

NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
authentication                             4.10.12      True              False                      False      35h     
baremetal                                   4.10.12       True             False                      False      35h     
etcd                                             4.10.12      True             False                      False      35h     
image-registry                             4.10.12       True            False                      False      35h     
ingress                                        4.10.12       True            False                      False      35h     
....

Reply:
I think clusteroperators would be in progressing/degraded state. so, as long as we have etcd and baremetal API in working order that should be enough to deploy the node. @zaneb can you comment on this?

Reply:
I think the idea to include this is very good (let's make this as resilient as possible). Also agreed with Rohan's suggestions or where it needs to go. Thanks guys!

Reply:
@zaneb would you agree?

Reply:
@pmannidi - I think I understand a little better, so would the command for step 1 be something like the following:

1. Ensure the pods in the machine-api namespace are in a `Running` status: 
`oc get pods -n openshift-machine-api`

Reply:
Yes, `oc get clusteroperator baremetal` should tell you whether everything is rolled out (unless CBO itself is down, though there is work underway in CVO to detect when its second-level operators are down).
It certainly can't hurt to check that, although tbh I'm not clear on which scenarios we are trying to prevent here. Is it that the metal3 Pod might be scheduled to the master that is down, but it won't be drained until after we delete the Machine? What should we tell users to do if the metal3 pod is _not_ ready? Just wait?

Reply:
Added a step to check the BMO availability.